tests/vmi_configuration_test.go

```diff
@@ -474,9 +474,13 @@ var _ = Describe("[sig-compute]Configurations", func() {
 		})
 
 		Context("[rfe_id:140][crit:medium][vendor:cnv-qe@redhat.com][level:component]with no memory requested", func() {
+			withoutRequests := func() libvmi.Option {
```

Comment:
If there is no explicit need for a closure function, it will be better not to use it.
Any chance you can move it to the tail of this file?

This is a nit, I will not insist.

Reply:
Done